src/execution_plan/optimizations/reduce_traversal.c

```diff
@@ -60,6 +60,7 @@ void reduceTraversal(ExecutionPlan *plan) {
 		   ae->operands[0].diagonal) continue;
 
 		/* Search to see if dest is already resolved */
+		if(op->childCount == 0) continue;
```

Comment:
We can reach this point when `op` is an Argument, and thus childless by definition. It might be better to block on `op->type` rather than `childCount`, and we may be able to do so earlier.